epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/LoggableHandler.java

```diff
@@ -40,4 +73,38 @@ public String logInvocation(Element element) {
         + elementAccess(element)
         + ", Epilogue.getConfig().errorHandler)";
   }
+
+  private String generateConditionalLogInvocation(
+      TypeElement reflectedType, Element element, List<TypeMirror> subtypes) {
+    StringBuilder builder = new StringBuilder(256);
+
+    // Combining multiple strings into fewer append calls
```

Comment:
This code runs on the programmer's computer, not the weak roboRIO. A few more append calls are negligible on performance; readability and maintainability should take higher priority